Dockerfile

```diff
@@ -11,61 +11,59 @@ LABEL \
     description="Mobile Security Framework is an intelligent, all-in-one open source mobile application (Android/iOS/Windows) automated pen-testing framework capable of performing static, dynamic analysis and web API testing"
 
 #Environment vars
-ENV DEBIAN_FRONTEND="noninteractive"
-ENV PDFGEN_PKGFILE="wkhtmltox_0.12.5-1.bionic_amd64.deb" 
-ENV PDFGEN_URL="https://github.com/wkhtmltopdf/wkhtmltopdf/releases/download/0.12.5/${PDFGEN_PKGFILE}"
-ENV JDK_FILE="openjdk-12_linux-x64_bin.tar.gz"
-ENV JDK_URL="https://download.java.net/java/GA/jdk12/GPL/${JDK_FILE}"
+ENV DEBIAN_FRONTEND="noninteractive" \
+    JDK_FILE="openjdk-12_linux-x64_bin.tar.gz" \
+    WKH_FILE="wkhtmltox-0.12.5-dev-163e124_linux-generic-amd64.tar.xz"
+
+ENV JDK_URL="https://download.java.net/java/GA/jdk12/GPL/${JDK_FILE}" \
+    WKH_URL="http://www.ajvg.com/downloads/${WKH_FILE}"
 
 #Update the repository sources list
 #Install Required Libs
 #see https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#run
 RUN apt update -y && apt install -y \
     build-essential \
+    git \
     libssl-dev \
     libffi-dev \
     libxml2-dev \
     libxslt1-dev \
     locales \
-    wget
-
+    sqlite3 \
+    fontconfig-config \
+    libjpeg-turbo8 \
+    libxrender1 \
+    libfontconfig1 \ 
+    libxext6 \
+    fontconfig \
+    xfonts-75dpi \
+    python3.6 \
+    python3-dev \
+    python3-setuptools \
+    wget && \
+    python3 /usr/lib/python3/dist-packages/easy_install.py pip
```

Comment:
There is no other documented way to install python3-pip?

Reply:
wouldn't `pip3` already be available?

Reply:
will change that, you're right

Reply:
pip is not installed by default